libphobos/libdruntime/gcc/unwind/arm.d

```diff
@@ -1,315 +1,64 @@
-/* GDC -- D front-end for GCC
-   Copyright (C) 2011, 2012 Free Software Foundation, Inc.
+// Exception handling and frame unwind runtime interface routines.
+// Copyright (C) 2011-2017 Free Software Foundation, Inc.
 
-   GCC is free software; you can redistribute it and/or modify it under
-   the terms of the GNU General Public License as published by the Free
-   Software Foundation; either version 3, or (at your option) any later
-   version.
+// GCC is free software; you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free
+// Software Foundation; either version 3, or (at your option) any later
+// version.
 
-   GCC is distributed in the hope that it will be useful, but WITHOUT ANY
-   WARRANTY; without even the implied warranty of MERCHANTABILITY or
-   FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
-   for more details.
+// GCC is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+// for more details.
 
-   You should have received a copy of the GNU General Public License
-   along with GCC; see the file COPYING3.  If not see
-   <http://www.gnu.org/licenses/>.
-*/
+// You should have received a copy of the GNU General Public License
+// along with GCC; see the file COPYING3.  If not see
+// <http://www.gnu.org/licenses/>.
 
-/* ARM unwind interface declarations for D.  This must match unwind-arm.h.  */
+// extern(C) interface for the ARM EABI unwinder library.
+// This corresponds to unwind-arm.h
 
 module gcc.unwind.arm;
 
 import gcc.config;
+
 static if (GNU_ARM_EABI_Unwinder):
 
-import gcc.builtins;
-import gcc.unwind.pe;
```

Comment:
Or this import. . I was mindful in testing this to make sure it's fine without this circular import. 

Apparently not.